File: Examen/Examenw/src/main/java/com/pe/edu/upeu/examenw/entity/OfertaTrabajo.java
package com.pe.edu.upeu.examenw.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDate;

@Entity
@Data
public class OfertaTrabajo {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String titulo;
    private String descripcion;
    private int experiencia;
    private LocalDate fechaFin;
    private LocalDate fechaInicio;
    private String formaAca;
    private String idiomas;
    private String puesto;
    private BigDecimal salario;
    private int vacantes;
    private String estado;
}

File: Proyecto/app/src/main/java/pe/edu/upeu/proyecto/OfertaTrabajo.kt
package pe.edu.upeu.proyecto

import java.math.BigDecimal

data class OfertaTrabajo(
    val id: Long,
    val titulo: String,
    val descripcion: String,
    val experiencia: Int,
    val fechaFin: String,
    val fechaInicio: String,
    val formaAca: String,
    val idiomas: String,
    val puesto: String,
    val salario: BigDecimal,
    val vacantes: Int,
    val estado: String
)
